src/Shared/Components/DeploymentConfigDiff/DeploymentConfigDiff.scss

```diff
@@ -0,0 +1,26 @@
+.deployment-config-diff {
+    display: grid;
+    grid-template-columns: 255px 1fr;
+    height: 100%;
+
+    &__accordion {
+        scroll-margin-top: 16px;
+    }
+
+    &__main-top {
+        padding-top: 45px;
+
+        &__header:first-child {
+            flex: 1 0 calc(50% - 2px);
+        }
+
+        &__header:last-child {
+            flex: 1 0 50%;
+        }
+    }
+
+    &__main-content {
+        height: calc(100vh - 122px);
+        overflow: auto;
+    }
```

Comment:
Suggestion: it would be great if we can get rid of calc here

Reply:
it is required